default/scripting/species/common/population.macros

```diff
@@ -173,7 +173,7 @@ PHOTOTROPHIC_BONUS
                 TargetPopulation low = 0
             ]
             accountinglabel = "VERY_BRIGHT_STAR"
-            priority = [[EARLY_POPULATION_PRIORITY]]
+            priority = [[EARLY_TARGET_POPULATION_POST_SCALING_PRIORITY]]
```

Comment:
As this depends on the TargetPopulation, should probably be given a later priority?

Reply:
As an additive adjustment post scaling, it doesn't need to be made later than other post-scaling additions, but as I noted in the Issue, your handling of this is fine as well.